src/main/java/competition/subsystems/turret/TurretSubsystem.java

```diff
@@ -86,17 +90,17 @@ public void turnLeft() {
     public void turnRight() {
         motor.simpleSet(-turnPowerProp.get());
     }
-
+    
     public void setPower(double power) {
         if (isCalibrated()) {
             // No sense running the protection code if we don't know where we are.
             
             // Check for any reason power should be constrained.
-            if (aboveMaximumAngle()) {
+            if (aboveMaximumAngle() || rightLimitStop()) {
                 // Turned too far left. Only allow right/negative rotation.
                 power = MathUtils.constrainDouble(power, -1, 0);
             }
-            if (belowMinimumAngle()) {
+            if (belowMinimumAngle() || leftLimitStop()) {
```

Comment:
consider renaming yours or belowMinimumAngle so they both talk about left/right or min/max angle but not a mix of them